uber/configspec.ini

```diff
@@ -70,15 +70,6 @@ groups_enabled = boolean(default=True)
 # made this a weekly rather than a daily email.
 check_placeholders = boolean(default=True)
 
```

Comment:
This stuff is moved into the other plugin's config, except for ``room_deadline`` which is just removed as a default option.  But the ``[dates]`` section intentionally can have extra deadlines thrown in with no downside, so our existing config for that will still work.